awscli/customizations/s3/subcommands.py

```diff
@@ -649,7 +665,7 @@ class CpCommand(S3TransferCommand):
             "or <S3Uri> <S3Uri>"
     ARG_TABLE = [{'name': 'paths', 'nargs': 2, 'positional_arg': True,
                   'synopsis': USAGE}] + TRANSFER_ARGS + \
-                [METADATA_DIRECTIVE, EXPECTED_SIZE, RECURSIVE]
+                [METADATA, METADATA_DIRECTIVE, EXPECTED_SIZE, RECURSIVE]
```

Comment:
By supporting it in sync, we're simply saying that we'll set metadata to whatever you've specified if we need to transfer a file to S3 right?  Seems like it would make sense.